src/components/SplashScreen/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { FC } from 'react'
+import Link from 'next/link'
+
+export const SplashScreen: FC = () => {
+  return (
+    <div className="flex flex-col justify-center h-full w-full">
+      <div className="bg-white p-6 max-w-xs md:max-w-none filter drop-shadow-lg rounded-lg md:min-w-xl md:w-1/2 mx-auto">
+        <div className="flex flex-row-reverse justify-between items-center md:flex-row md:justify-start">
+          <svg
+            width="67"
+            height="67"
+            viewBox="0 0 67 67"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+            className="mr-4"
+          >
+            <path
+              d="M33.5 3.125V6.5V3.125ZM33.5 60.5V63.875V60.5ZM63.875 33.5H60.5H63.875ZM6.5 33.5H3.125H6.5ZM54.9785 54.9785L52.5924 52.5924L54.9785 54.9785ZM14.4076 14.4076L12.0215 12.0215L14.4076 14.4076ZM54.9785 12.0215L52.5924 14.4076L54.9785 12.0215ZM14.4076 52.5924L12.0215 54.9785L14.4076 52.5924ZM47 33.5C47 37.0804 45.5777 40.5142 43.0459 43.0459C40.5142 45.5777 37.0804 47 33.5 47C29.9196 47 26.4858 45.5777 23.9541 43.0459C21.4223 40.5142 20 37.0804 20 33.5C20 29.9196 21.4223 26.4858 23.9541 23.9541C26.4858 21.4223 29.9196 20 33.5 20C37.0804 20 40.5142 21.4223 43.0459 23.9541C45.5777 26.4858 47 29.9196 47 33.5V33.5Z"
+              stroke="#333333"
+              strokeWidth="5"
+              strokeLinecap="round"
+              strokeLinejoin="round"
+            />
+          </svg>
+          <div>
+            <h1 className="font-bold text-2xl">30°C und kein Schatten?</h1>
+            <h1 className="font-bold text-2xl text-layer-turquoise-300">
```

Comment:
That second part should be a `span` inside the first `h1`, because there should never be two `h1`s on the same page.